website/static/schema.json

```diff
@@ -59,8 +59,11 @@
           "type": "string"
         },
         "prompt": {
-          "description": "A prompt that will be presented before a task is run. Declining will cancel running the current and any subsequent tasks.",
-          "type": "string"
+          "description": "One or more prompts that will be presented before a task is run. Declining will cancel running the current and any subsequent tasks.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
         },
```

Comment:
Changing the schema like that is a breaking change.
You need to keep the old type. You can use `oneOf` property to define two types

Otherwise, we won't be able to validate current Taskfile (see : 
![image](https://github.com/user-attachments/assets/2317f360-f47e-4944-adea-d2b0ec0b3182)
)